package.json

```diff
@@ -34,6 +35,7 @@
     "dist"
   ],
   "dependencies": {
+    "@types/json-schema": "^7.0.3",
```

Comment:
Not sure it is good idea. Some projects doesn't use ts so they don't need extra libraries in deps